frontend/src/toolbar/posthog.ts

```diff
@@ -4,10 +4,12 @@ const DEFAULT_API_KEY = 'sTMFPsFhdP1Ssg'
 
 const runningOnPosthog = !!window.POSTHOG_APP_CONTEXT
 const apiKey = runningOnPosthog ? window.JS_POSTHOG_API_KEY : DEFAULT_API_KEY
-const apiHost = runningOnPosthog ? window.JS_POSTHOG_HOST : 'https://app.posthog.com'
+const apiHost = runningOnPosthog ? window.JS_POSTHOG_HOST : 'https://d1iyf54ud053a4.cloudfront.net'
```

Comment:
Not sure how I feel about having a cloudfront domain here...

Reply:
That's what  @danielxnj set up for the reverse proxy. Can you explain a bit more about your hesitation and maybe suggest an alternative?

Reply:
I think the two main things are branding/optics and mainly (for me) potential CORS issues since that's a totally different root domain.

Reply:
Make sense, seems like Daniel was able to get an internal domain pointed at the cloudfront one!